README.md

```diff
@@ -46,7 +46,7 @@ sdk.start().then(() => {
 | metricEndpoint | OTEL_EXPORTER_OTLP_METRIC_ENDPOINT | n        | https://ingest.lightstep.com:443/metrics           |
 | accessToken    | LS_ACCESS_TOKEN                    | n        | -                                                  |
 | logLevel       | OTEL_LOG_LEVEL                     | n        | info                                               |
-| propagators    | OTEL_PROPAGATORS                   | n        | b3                                                 |
+| propagators    | OTEL_PROPAGATORS                   | n        | b3 - now also available "b3single"                 |
```

Comment:
I think we should move this comment to the changelog as this table is for documenting the defaults and not all acceptable options.